### PR TITLE
Update custom-notifications.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information that "Android Enterprise personally-owned with a work profile" supports custom notifications, but currently "Android Enterprise fully managed" does not.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2788

**Comment by PRMerger3** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/remote-actions/custom-notifications.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ ms.collection: M365-identity-device-management
 
 # Send custom notifications in Intune
 
-Use Microsoft Intune to send custom notifications to the users of managed iOS/iPadOS and Android devices. These messages appear as standard push notifications from the Company Portal app and from the Microsoft Intune app on a user's device, just as notifications from other applications on the device appear. Intune custom notifications aren't supported by macOS and Windows devices.
+Use Microsoft Intune to send custom notifications to the users of managed iOS/iPadOS and Android devices. These messages appear as standard push notifications from the Company Portal app and from the Microsoft Intune app on a user's device, just as notifications from other applications on the device appear. Intune custom notifications aren't supported by macOS and Windows devices. Android Enterprise personally-owned with a work profile supports custom notifications, but currently Android Enterprise fully managed does not.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/custom-notifications.md:33]:
```suggestion
Use Microsoft Intune to send custom notifications to the users of managed iOS/iPadOS and Android devices. These messages appear as standard push notifications from the Company Portal app and from the Microsoft Intune app on a user's device, just as notifications from other applications on the device appear. Intune custom notifications aren't supported by macOS and Windows devices. Android Enterprise personally-owned with a work profile supports custom notifications, but currently, Android Enterprise fully managed does not. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger19** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Review by Smritib17 — CHANGES_REQUESTED** [review]
Based on inputs from the PM I gather that AE fully managed devices do support custom notifications. However, the notification may not have showed up for the customer because of some reliability issue with custom notifications. 
This is documented here: [Send custom notifications to users with Microsoft Intune - Microsoft Intune | Microsoft Docs.](https://docs.microsoft.com/en-us/mem/intune/remote-actions/custom-notifications)
- _Intune, the Company Portal app, and the Microsoft Intune app, can't guarantee delivery of a custom notification. Custom notifications might show up after several hours of delay, if at all, so they shouldn't be used for urgent messages._

**Comment by msbemba** [comment]
@Smritib17  Thank you for the clarification.

**Pull request closed without merging**